lib/__tests__/overrides.test.js

```diff
@@ -208,6 +208,7 @@ describe('single input file. all overrides are matching', () => {
 			files: [path.join(fixturesPath, 'style.scss')],
 			config: {
 				extends: ['./at-rule-no-unknown.json'],
+				customSyntax: 'postcss-scss',
```

Comment:
I think it is better to add it to https://github.com/stylelint/stylelint/blob/main/lib/__tests__/fixtures/config-overrides/scss.json instead. Because it makes more sense to set custom syntax for `*.scss` files.

Reply:
fixed in #6645